FNPlugin/Refinery/SolarWindProcessor.cs

```diff
@@ -86,24 +91,26 @@ public SolarWindProcessor(Part part)
         protected double _maxCapacityHelium4Mass;
         protected double _maxCapacityMonoxideMass;
         protected double _maxCapacityNitrogenMass;
+        protected double _maxCapacityNeonMass;
 
         protected double _availableSolarWindMass;
         protected double _spareRoomHydrogenMass;
         protected double _spareRoomHelium3Mass;
         protected double _spareRoomHelium4Mass;
         protected double _spareRoomMonoxideMass;
         protected double _spareRoomNitrogenMass;
+        protected double _spareRoomNeonMass;
 
         /* these are the constituents of solar wind with their appropriate mass ratios. According to http://solar-center.stanford.edu/FAQ/Qsolwindcomp.html and other sources,
-         * about 90 % of atoms in solar wind are hydrogen. About 8 % is helium (he-3 is less stable than he-4 so i guesstimated that about quarter of those 8 % could be he-3,
-         * so I take that 2 % of atoms in solar wind are he-3 and 6 % of atoms in solar wind are the more stable he-4). There are supposedly only trace amounts of heavier elements such as C, N and O,
-         * so I decided on 1 % of atoms for both CO and N. The exact fractions were calculated as (percentage * atomic mass of the element) / Sum of (percentage * atomic mass of the element) for every element.
+         * about 90 % of atoms in solar wind are hydrogen. About 8 % is helium (he-3 is less stable than he-4 and apparently the He-3/He-4 ratio is very close to 1/2000), so that's about 7,996 % He-4 and 0.004 % He-3. There are supposedly only trace amounts of heavier elements such as C, O, N and Ne,
+         * so I decided on 1 % of atoms for CO and 0.5 % for Nitrogen and Neon. The exact fractions were calculated as (percentage * atomic mass of the element) / Sum of (percentage * atomic mass of the element) for every element.
         */
-        protected double _hydrogenMassByFraction = 0.1118; // see above how I got this number (as an example 0.9 * 1.008 / 8.1133 = 0.1118), ie. percentage times atomic mass of He-3 / sum of the same stuff in numerator for every element
-        protected double _helium3MassByFraction = 0.0743; // because examples are nice to have (0.2 * 3.016 / 8.1133 = 0.0743)
-        protected double _helium4MassByFraction = 0.2959;
-        protected double _monoxideMassByFraction = 0.3452;
-        protected double _nitrogenMassByFraction = 0.1726;
```

Comment:
Dammit, I completely forgot about Tritium. According to NASA, the ratio for tritium in solar wind is "H-3/H-1 limit of 2 times 10 to the minus eleventh power" (or 2 * 10^-11 or 0,00000000002 or 1 in 50,000,000,000). Should I add it?